src/adapter/src/catalog/open.rs

```diff
@@ -1121,7 +1121,7 @@ impl Catalog {
             match state.insert_system_configuration(&name, VarInput::Flat(&value)) {
                 Ok(_) => (),
                 Err(AdapterError::VarError(VarError::UnknownParameter(name))) => {
-                    warn!(%name, "cannot load unknown system parameter from catalog storage");
+                    warn!(%name, "cannot load unknown system parameter from catalog storage to set configured parameter:");
```

Comment:
nit: remove the `:` at the end. The format used for the `warn!` is structured logging not printf-style string interpolation. The log line will be a json string which will look as follows:

```json
{ "level": "info", "name": "$name", "message": "cannot load unknown system parameter from catalog storage to set configured parameter:" }
```